akka-docs/rst/java/stream/stages-overview.rst

```diff
@@ -713,6 +735,57 @@ a function has to be provided to calculate the individual cost of each element.
 
 **completes** when upstream completes
 
+intersperse
+^^^^^^^^^^^
+Intersperse stream with provided element similar to ``List.mkString``. It can inject start and end marker elements to stream.
+
+**emits** when upstream emits an element or before with the `start` element if provided
+
+**backpressures** when downstream backpressures
+
+**completes** when upstream completes
+
+limit
+^^^^^
+Limit number of element from upstream to given ``max`` number.
+
+**emits** when upstream emits and the number of emitted elements has not reached max
+
+**backpressures** when downstream backpressures
+
+**completes** when upstream completes and the number of emitted elements has not reached max
+
+limitWeighted
+^^^^^^^^^^^^^
+Ensure stream boundedness by evaluating the cost of incoming elements using a cost function.
+Evaluated cost of each element defines how many elements will be allowed to travel downstream.
+
+**emits** when upstream emits and the number of emitted elements has not reached max
+
+**backpressures** when downstream backpressures
+
+**completes** when upstream completes and the number of emitted elements has not reached max
+
+log
+^^^
+Log elements flowing through the stream as well as completion and erroring.
```

Comment:
maybe mention that the log level can be set with the attributes `Attributes.loglevels` (scala) and `Attributes.createLogLevels()` (java) ?